functions/startJob.js

```diff
@@ -0,0 +1,119 @@
+const AWS = require('aws-sdk');
+const https = require('https');
+const url = require('url');
+const initAppInsights = require('./appInsights.js');
+
+const client = new AWS.SecretsManager();
+
+async function getSecret(secretId, callback) {
+    return new Promise((resolve, reject) => {
+        client.getSecretValue({ SecretId: secretId }, function (err, data) {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(data.SecretString);
+            }
+        });
+    });
+}
+
+async function startJob(startJobUrl, releaseKey, jobInputArguments, tenantName, folderId, access_token) {
+    let startJobData = {
+        startInfo: {
+            ReleaseKey: releaseKey,
+            Strategy: "JobsCount",
```

Comment:
Let's add a parameter to the lambda that takes classic or Modern and defaults to classic.  We need to point to documentation on folders in the parameter description.

Reply:
Added a param `folderType` which defaults to `classic`. Values can be 'classic'/'modern'. Case insensitive comparison is done here to avoid any issue.